lib/msal-common/src/config/PublicClientSPAConfiguration.ts

```diff
@@ -5,14 +5,14 @@
 import { Configuration, buildConfiguration } from "./Configuration";
 
 /**
- * @type AuthOptions: Use this to configure the auth options in the Configuration object
+ * @type SPAAuthOptions: Use this to configure the auth options in the Configuration object
  *
  *  - clientId                    - Client ID of your app registered with our Application registration portal : https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredAppsPreview in Microsoft Identity Platform
  *  - authority                   - You can configure a specific authority, defaults to " " or "https://login.microsoftonline.com/common"
  *  - redirectUri                 - The redirect URI of the application, this should be same as the value in the application registration portal.Defaults to `window.location.href`.
  *  - postLogoutRedirectUri       - Used to redirect the user to this location after logout. Defaults to `window.location.href`.
  */
-export type AuthOptions = {
+export type SPAAuthOptions = {
```

Comment:
postLogoutRedirectUri? It's in the block comment above but not in the options below

Reply:
It's on line 19.

Reply:
My b. Didn't see it as github collapsed a bunch of lines :)